gymnasium/core.py

```diff
@@ -16,23 +16,18 @@
 import numpy as np
 
 from gymnasium import spaces
-from gymnasium.logger import warn
 from gymnasium.utils import seeding
 
 if TYPE_CHECKING:
     from gymnasium.envs.registration import EnvSpec
 
-if sys.version_info[0:2] == (3, 6):
```

Comment:
Is this PR doubling for 3.6 removal?

Reply:
It wasn't intended to but I was doing both at the same time. Will revert